specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.6-preview.1/backuprestore.json

```diff
@@ -113,7 +113,7 @@
               "Retry-After": {
                 "description": "The recommended number of seconds to wait before calling the URI specified in Azure-AsyncOperation.",
                 "type": "integer",
-                "format": "int64"
+                "format": "int32"
```

Comment:
Thanks for making this change.

I wonder, is it necessary to specify the format for this header? 
I see on [lines 58-61](https://github.com/Azure/azure-rest-api-specs/pull/30115/files#diff-4c203dfce12178b8c7e6ba4dda9615a0d4ae836981d2ea76f84cc40362037ec7R58-R61), where the same header has been defined, the format is not specified.

Reply:
It's better to be explicit. Most of the `Retry-After` headers herein are.